### PR TITLE
moved loading of groupedinv order to sync

### PR DESCRIPTION
**Pull request by y-golde**
fixes [1484](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1484/)

#1272 solved half of the problem - what was happening is that when the investigation was being transferred `fetchTableData()` and `fetchInvestigationsByGroupId()` were running in parallel - both using global 'isLoading' - meaning that if both started at the same time - the one that was the quickest would have caused the loading to stop (even though the other one was still on-going)

so I changed the order of those actions to sync - first of all  `fetchInvestigationsByGroupId` runs in parallel on all of the group ids and when its done `fetchTableData` is run.

this also solved the problem of changing multiple grouped investigations loading (by the footer).

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -419,6 +415,8 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                                 const currentGroupedInvestigationsLength = allGroupedInvestigations.get(indexedRow.groupId)?.length! - 1; // not including row head
                                                 const indexedGroupedInvestigationRow = convertToIndexedRow(row);
                                                 const isGroupedRowClickable = isInvestigationRowClickable(row.mainStatus);
+
+                                                console.log(row.investigationDesk);
```
Comment (chenFriedman) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:419]: remove console

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 2]
```diff
@@ -372,7 +368,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                 const indexedRow = convertToIndexedRow(row);
                                 const isRowClickable = isInvestigationRowClickable(row.mainStatus);
                                 const isGroupShown = checkGroupedInvestigationOpen.includes(indexedRow.epidemiologyNumber);
-
+
```
Comment (chenFriedman) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:371]: remove space

**Review by chenFriedman — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 2 above)
Comment (y-golde) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1284/commits/66071bfa273b9fc1d850578ba49cc35c064bba2f)

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx]: fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1284/commits/66071bfa273b9fc1d850578ba49cc35c064bba2f)

**Review by Noam6421 — APPROVED** [review]

**Review by chenFriedman — APPROVED** [review]

**Pull request merged**